smart_dog_app/src/containers/GoogleMapsPage.jsx

```diff
@@ -1,137 +1,160 @@
-import React, { Component } from "react";
```

Comment:
I don't think you meant to comment out the entire page. Also, you don't have to close and reopen your pull request to update things. You can simply commit the new changes to your own branch and I can see the updates here when I refresh.